<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDissimilarityMatrixDlg</class>
 <widget class="QDialog" name="OpenDissimilarityMatrixDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open dissimilarity matrix</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dissimilarity matrix:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtDissimilarityMatrix">
       <property name="toolTip">
        <string>Matrix indicating dissimilarity between sequences</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnOpenDissimilarityMatrix">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Metadata file (optional):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtMetadataFile">
       <property name="toolTip">
        <string>Indicates metadata for each sample site</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnOpenMetadataFile">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadData">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoadData</sender>
   <signal>clicked()</signal>
   <receiver>OpenDissimilarityMatrixDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>OpenDissimilarityMatrixDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
